templates/prescription/general_edit.html

```diff
@@ -99,204 +86,240 @@
-    <select class="input-sm" name="form" id="form">{html_options options=$prescription->form_array selected=$prescription->form}</select>
-    <select class="input-sm" name="route" id="route">{html_options options=$prescription->route_array selected=$prescription->route}</select>
-    <select class="input-sm" name="interval" id="interval">{html_options options=$prescription->interval_array selected=$prescription->interval}</select>
-{/if}
-  </td>
-</tr>
-<tr>
-  <td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xlt t='Refills'}</td>
-  <td COLSPAN="2" class="text" ALIGN="LEFT" VALIGN="MIDDLE" >
-    {html_options name="refills" options=$prescription->refills_array selected=$prescription->refills}
-{if $SIMPLIFIED_PRESCRIPTIONS}
-    <input TYPE="hidden" ID="per_refill" NAME="per_refill" VALUE="{$prescription->per_refill|attr}" />
-{else}
-    &nbsp; &nbsp; # {xlt t='of tablets'}:
-    <input class="input-sm" TYPE="TEXT" ID="per_refill" NAME="per_refill" SIZE="2" MAXLENGTH="10" VALUE="{$prescription->per_refill|attr}" />
-{/if}
-  </td>
-</tr>
-<tr>
-  <td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xlt t='Notes'}</td>
-  <td COLSPAN="2" class="text" ALIGN="LEFT" VALIGN="MIDDLE" >
-  <textarea class="form-control" name="note" cols="30" rows="1" wrap="virtual">{$prescription->note|text}</textarea>
-  </td>
-</tr>
-<tr>
-{if $WEIGHT_LOSS_CLINIC}
-  <td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xlt t='Substitution'}</td>
-  <td COLSPAN="2" ALIGN="LEFT" VALIGN="MIDDLE" >
-    {html_options name="substitute" options=$prescription->substitute_array selected=$prescription->substitute}
-  </td>
-{else}
-  <td COLSPAN="1" class="text input-sm" ALIGN="right" VALIGN="MIDDLE" >{xlt t='Add to Medication List'}</td>
-  <td COLSPAN="2" class="text input-sm" ALIGN="LEFT" VALIGN="MIDDLE" >
-    {html_radios class="input-sm" name="medication" options=$prescription->medication_array selected=$prescription->medication}
-    &nbsp; &nbsp;
-    {html_options class="input-sm" name="substitute" options=$prescription->substitute_array selected=$prescription->substitute}
-  </td>
-{/if}
-</tr>
-</table>
-<input type="hidden" name="id" value="{$prescription->id|attr}" />
-<input type="hidden" name="process" value="{$PROCESS|attr}" />
-<script language='JavaScript'>
-{$ENDING_JAVASCRIPT}
-</script>
-</form>
+        {if $SIMPLIFIED_PRESCRIPTIONS && !$prescription->size}
+            {assign var="dosageVisibility" value="hidden"}
+        {else}
+            {assign var="dosageVisibility" value=""}
+        {/if}
+        <div class="form-group {$dosageVisibility}">
+            <label for="size" class="control-label col-sm-2">{xlt t="Dosage"}</label>
+            <div class="col-sm-2">
+                <input type="text" class="form-control" name="size" id="size" placeholder="{xlt t='ex 81'}" maxlength="10" value="{$prescription->size|attr}">
```

Comment:
use xla instead of xlt when within an html attribute. several other places to do this in this script.